src/services/ats/useRepoATS.tsx

```diff
@@ -0,0 +1,114 @@
+import { useQuery } from '@tanstack/react-query'
+import { z } from 'zod'
+
+import {
+  RepoNotFoundErrorSchema,
+  RepoOwnerNotActivatedErrorSchema,
+} from 'services/repo'
+import Api from 'shared/api'
+import A from 'ui/A'
+
+const RepositorySchema = z.object({
+  __typename: z.literal('Repository'),
+  isATSConfigured: z.boolean().nullish(),
```

Comment:
Can these be undefined or just null? If so might be better to use `nullable`() instead